templates/phoenix_elixir/files/config/prod.secret.exs

```diff
@@ -0,0 +1,26 @@
+# In this file, we load production configuration and secrets
+# from environment variables. You can also hardcode secrets,
+# although such is generally not recommended and you have to
+# remember to add this file to your .gitignore.
+use Mix.Config
+
+secret_key_base =
+  System.get_env("PLATFORM_PROJECT_ENTROPY") ||
```

Comment:
This is reading from the original Platform variable, but in .platform.app.yaml it's being mapped over to the default variable name Elixir uses.  Why do both?

Reply:
To support local dev, reading from the default env var but mapping the Psh value to it seems like the better option.  Viz, have this line read from the original default variable name.

Reply:
👍